apps/docs/src/app/core/component-docs/bar/examples/bar-custom-colors-example.component.scss

```diff
@@ -0,0 +1,9 @@
+.fd-bar {
+    background-color: var(--sapShellColor);
+    padding: 0 2rem;
+}
+fd-bar-element {
+    color: var(--sapShell_TextColor);
+    font-weight: bold;
+    vertical-align: middle;
+}
```

Comment:
these should be moved to fund-styles. Please open a PR these as well and add here a comment on top of the file to remove after fund-styles release

Reply:
I agree with @mikerodonnell89 . This is just an example how the applications can build their shellbar with custom components.